src/recorder.js

```diff
@@ -40,9 +43,7 @@ var Recorder = function( config ){
 };
 
 Recorder.isRecordingSupported = function(){
-  var AudioContext = global.AudioContext || global.webkitAudioContext;
-  var getUserMedia = global.navigator && ( global.navigator.getUserMedia || ( global.navigator.mediaDevices && global.navigator.mediaDevices.getUserMedia ) );
-  return AudioContext && getUserMedia;
+  return (global.AudioContext || global.webkitAudioContext) && getUserMedia.isSupported;
```

Comment:
`getUserMedia.isSupported` is breaking the unit tests (which are run in a node env)